bin/pre-commit

```diff
@@ -30,6 +30,6 @@ BRANCH_NAME=$(git branch | grep '*' | sed 's/* //')
 if [ $BRANCH_NAME != '(no branch)' ]; then
   cd $ROOT
 
-  echo "formatting..."
-  bin/fmt.sh
+#  echo "formatting..."
```

Comment:
These scripts failed with this work space. some directories doesn't exist yet. If I hook the original pre-commit,  'git commit' will always fail.